flake.nix

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I propose adding credit to original author: cc @SharzyL to the nix magic.

Reply:
Would like to hear from @SharzyL how this should be added.
e.g. what identity should be used?

Reply:
Referring as “Sharzy” should be fine. For example adding a credit to t1 project and me.